src/App/mermaidData/fishNameHelpers.jsx

```
@@ -57,49 +87,29 @@ export const getFishNameOptions = ({ species, genera, families, groupings = [] }
   return [...speciesOptions, ...generaAndFamiliesOptions]
 }
 
-export const getFishNameTable = ({
+const FishNameTableContent = ({
   fishFamilies,
```

ESLint

[error] 'fishFamilies' is missing in props validation

(react/prop-types)